README.md

```diff
@@ -1,19 +1,13 @@
 Chitter Challenge
```

Comment:
would be good to have instructions on how to set up the database and some screenshots etc. of how the website works